faqs/galaxy/datasets_BAM_dataset_identifiers.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Finding BAM dataset identifiers
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+Quickly learn what the identifiers are in any **BAM** dataset that is the result from mapping.
```

Comment:
this looks to me like a something that could go to the "description" field in the metadata? See [this example](https://github.com/galaxyproject/training-material/blob/main/faqs/galaxy/analysis_troubleshooting.md).